server/routes/addPrisoner/fields.ts

```diff
@@ -13,8 +13,8 @@ const fields = {
     },
     items: [
       {
-        text: 'Please select one',
-        selected: true,
+        text: 'overwritten during runtime',
```

Comment:
These are set in the controllers. So I have a function in the controller `setOptions` which imports the values from the constants in the report configurations you made so that we don't need to manually keep the field options maintained. It reads them in from there and sets the options as part of the page being generated.